Automattic/jetpack
# Admin: do not display upgrade messages when on offline mode

#### Changes proposed in this Pull Request:
﻿We already display a generic Offline mode message for most of our AAG cards. Let's display the same message for Akismet and VideoPress.

**Before**

![image](https://user-images.githubusercontent.com/426388/137721032-cb10a37d-5c22-4b22-b416-d769eba1ce61.png)


**After**

![image](https://user-images.githubusercontent.com/426388/137720906-2be3bc9f-95b7-47db-ba15-9f3ad746c329.png)


#### Jetpack product discussion

* N/A

#### Does this pull request change what data or activity we track or use?

* No

#### Testing instructions:

* Start from a brand new site.
* Add `define( 'JETPACK_DEV_DEBUG', true );` to your site's `wp-config.php` file.
* Go to Jetpack > Dashboard.
* You should not see links to upgrade to a plan anymore, since your site is not connected and you cannot purchase a plan.
* Remove the constant and connect your site to your WordPress.com account, without purchasing a plan. The prompts should be displayed properly (and an invitation to enable VideoPress).
